src/main/java/xyz/kyngs/herbot/handlers/command/argument/arguments/OptionArgument.java

```diff
@@ -0,0 +1,26 @@
+package xyz.kyngs.herbot.handlers.command.argument.arguments;
+
+import net.dv8tion.jda.api.JDA;
+import xyz.kyngs.herbot.handlers.command.argument.AbstractArgument;
+
+public class OptionArgument extends AbstractArgument {
+    public OptionArgument(String... optionsOrDescription) {
+        super("příkaz", optionsOrDescription);
+    }
+
+    @Override
+    public Object parseArg(String arg, JDA jda) {
+        var ok = false;
+        for (String s : optionsOrDescription) {
+            if (s.contentEquals(arg)) {
+                ok = true;
+                break;
+            }
+        }
+        if (ok) {
+            return arg;
+        } else {
+            throw new IllegalArgumentException();
+        }
+    }
```

Comment:
This could be rewritten as
`    @Override
    public Object parseArg(String arg, JDA jda) {
        var ok = false;
        for (String s : optionsOrDescription) {
            if (s.contentEquals(arg)) {
                return arg;
            }
        }
        throw new IllegalArgumentException();
    }`
    which simplifies the code, but this form is still ok.